<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Cloud Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QOpenGLWidget" name="openGLWidget"/>
    </item>
    <item>
     <widget class="QWidget" name="controlPanel">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save Video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="frameSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="frameLabel">
         <property name="text">
          <string>Frame: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cameraComboBox">
         <property name="currentText">
          <string>Top</string>
         </property>
         <item>
          <property name="text">
           <string>Top</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Front</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Back</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pointSizeSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fpsSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="axesCheckBox">
         <property name="text">
          <string>Show Axes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorModeComboBox">
         <property name="currentText">
          <string>Intensity</string>
         </property>
         <item>
          <property name="text">
           <string>Intensity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Color</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Height</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cameraXSpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cameraYSpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cameraZSpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets/>
 <resources/>
 <connections/>
</ui>
